packages/@aws-cdk/aws-appmesh/lib/virtual-router.ts

```diff
@@ -43,7 +43,7 @@ export interface VirtualRouterBaseProps {
    *
    * @default - A listener on HTTP port 8080
    */
-  readonly listener?: Listener;
+  readonly listeners?: VirtualRouterListener[];
```

Comment:
Still missing a second BREAKING CHANGE in the PR description for this change. Take a look here: https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md#step-4-commit on how to format multiple breaking changes in a single PR.